build.sbt

```diff
@@ -11,13 +14,14 @@ lazy val microservice = Project(appName, file("."))
     scalaVersion := "2.13.12",
     scalacOptions ++= Seq("-feature", "-language:postfixOps", "-Wconf:src=routes/.*:s"),
     libraryDependencies ++= AppDependencies(),
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
     PlayKeys.playDefaultPort := 9261
   )
   .settings(scoverageSettings)
-  .configs(IntegrationTest)
-  .settings(integrationTestSettings())
+
+lazy val it = project
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(DefaultBuildSettings.itSettings())
 
 lazy val scoverageSettings =
```

Comment:
Was told we agreed to move this into a separate file like this
https://github.com/hmrc/individual-employment/blob/09d40f09289c82419eb5eadd6c80659e6db9d7d8/project/CodeCoverageSettings.scala